pkg/config/testdata/tracee-config.yml

```diff
@@ -0,0 +1,57 @@
+kind: TraceeConfig
```

Comment:
What about supporting multiple config files? This file can very quickly turn hugh and unorganized like the falco signatures - which is best avoided in my opinion

Reply:
BTW I think it will be better to call it something like rules-config or just rules.yml because we will have an actual config for tracee.

Reply:
My idea was to define a single config that's read by tracee-ebpf and tracee-rules to keep it comprehensive and in one place. In particular, event selectors will probably be read by tracee-epbf to implement in kernel filtering. The kind and apiVersion are common metadata attributes to version configuration schema.

Reply:
Anyway, I don't have strong opinion on a single (potentially huge) file vs multiple smaller YAML chunks. To be discussed.

Reply:
The current behaviour implemented in this PR is as follows: Tracee-Rules loads 0 to many *.yaml files from the --rules-dir directory. Each file can define 0 to many CEL signatures. In other words, you can have 1 files with N signatures or N files with 1 signature or, in general, N file with M signatures.